integration/src/main/java/scripts/testscripts/RemoveUser.java

```diff
@@ -25,24 +26,23 @@
 import java.util.List;
 import java.util.UUID;
 import org.apache.commons.lang3.RandomStringUtils;
-import scripts.utils.BqDataTableUtils;
-import scripts.utils.BqDatasetUtils;
-import scripts.utils.ClientTestUtils;
-import scripts.utils.CloudContextMaker;
-import scripts.utils.GcsBucketObjectUtils;
-import scripts.utils.GcsBucketUtils;
-import scripts.utils.NotebookUtils;
-import scripts.utils.WorkspaceAllocateTestScriptBase;
+import org.broadinstitute.dsde.workbench.client.sam.api.GroupApi;
+import scripts.utils.*;
 
 public class RemoveUser extends WorkspaceAllocateTestScriptBase {
 
   private TestUserSpecification privateResourceUser;
   private TestUserSpecification sharedResourceUser;
   private String projectId;
+  private String groupEmail;
+  private String groupName;
   private CreatedControlledGcpGcsBucket sharedBucket;
   private CreatedControlledGcpGcsBucket privateBucket;
   private GcpBigQueryDatasetResource privateDataset;
   private CreatedControlledGcpAiNotebookInstanceResult privateNotebook;
+  private GroupApi samGroupApi;
```

Comment:
Usually we initialize these in setup, not a test.

I would just delete this and define inline. If you have `samGroupApi`, you can't tell which user it's for.

Delete line 43 and change 64 to:

```
GroupApi samGroupApi = SamClientUtils.samGroupApi(testUsers.get(1), server);
```

Change 140 to:

```
GroupApi samGroupApi = SamClientUtils.samGroupApi(testUsers.get(1), server);
SamRetry.retry(() -> samGroupApi.deleteGroup(groupName));
```

Reply:
Hi Melissa. I also feel not really good while I wrote it. The issue is the `testUsers` in the doSetup will also need to transfer to the `doUserJourney`. and then it will be more complicated. Do you have any suggestions on how to avoid it?